### DISCUSSION TITLE
Call to a member function prepare() on null Laravel 9, PHP 8 with MongoDB connection

### DISCUSSION THREAD

**Discussion by sts-ryan-holton** [spatie/laravel-schedule-monitor#78] [Q&A]
I've just installed version 3.0.3 of this package into a reasonable fresh Laravel 9 application running on PHP 8.0, I'm using MongoDB as my database and have set up my connection to my database using the [laravel-mongodb](https://github.com/jenssegers/laravel-mongodb) integration.

My database works and can save and retrieve data to it as I have a few other models set up, it's worth noting that I had to change the `$table` references to `$collection` in your database migrations that I've migrated into my project as this is what's needed for this integration.

Now, when I run `php artisan schedule-monitor:sync` I get an error thrown:

>  Call to a member function prepare() on null

What might this mean?

![Screenshot 2022-05-27 at 13 45 20](https://user-images.githubusercontent.com/25177538/170702260-d6563472-8038-41bb-99d1-188ccbad67c9.png)

**Comment by freekmurze** [comment]
We don't support Mongo. If this is easily fixed, I'd accept a PR.
